2024-12-11 03:53:58,436 - INFO - Finished! Total sfe time: 10.41217589378357 
2024-12-11 03:53:58,437 - DEBUG - 912,server, Parsed server ram usage: 80392
2024-12-11 03:53:58,437 - DEBUG - 912,client, Parsed client ram usage: 81216
2024-12-11 03:54:21,402 - DEBUG - client all values : ['modprobe cpufreq_stats failedLoaded 0 prior measurements', 'RAPL device for cpu 0', 'RAPL device for cpu 0', 'Devfreq not enabled', 'glob returned GLOB_ABORTED', 'Preparing to take measurements', 'Taking 1 measurement(s) for a duration of 11 second(s) each.', '2024-12-11 03:52:29,982 - tno.mpc.communication.httphandlers - INFO - Serving on 192.168.1.1:61002', '2024-12-11 03:52:29,985 - tno.mpc.communication.httphandlers - ERROR - Message not received.', 'Traceback (most recent call last):', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1061, in _wrap_create_connection', '    sock = await aiohappyeyeballs.start_connection(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohappyeyeballs/impl.py", line 103, in start_connection', '    raise first_exception', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohappyeyeballs/impl.py", line 81, in start_connection', '    sock = await _connect_sock(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohappyeyeballs/impl.py", line 165, in _connect_sock', '    await loop.sock_connect(sock, address)', '  File "/usr/lib/python3.9/asyncio/selector_events.py", line 502, in sock_connect', '    return await fut', '  File "/usr/lib/python3.9/asyncio/selector_events.py", line 537, in _sock_connect_cb', "    raise OSError(err, f'Connect call failed {address}')", "ConnectionRefusedError: [Errno 111] Connect call failed ('192.168.1.2', 61001)", '', 'The above exception was the direct cause of the following exception:', '', 'Traceback (most recent call last):', '  File "/home/kamil/.local/lib/python3.9/site-packages/tno/mpc/communication/httphandlers.py", line 193, in _send', '    async with self.session.post(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/client.py", line 1355, in __aenter__', '    self._resp: _RetType = await self._coro', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/client.py", line 659, in _request', '    conn = await self._connector.connect(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 557, in connect', '    proto = await self._create_connection(req, traces, timeout)', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1002, in _create_connection', '    _, proto = await self._create_direct_connection(req, traces, timeout)', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1336, in _create_direct_connection', '    raise last_exc', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1305, in _create_direct_connection', '    transp, proto = await self._wrap_create_connection(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1076, in _wrap_create_connection', '    raise client_error(req.connection_key, exc) from exc', "aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host 192.168.1.2:61001 ssl:default [Connect call failed ('192.168.1.2', 61001)]", '2024-12-11 03:52:31,048 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-11 03:52:31,080 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-11 03:52:31,160 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-11 03:52:31,207 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', 'TP is  0.001', '2024-12-11 03:52:31,222 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-11 03:52:31,241 - tno.mpc.communication.httphandlers - INFO - HTTPServer: Shutting down server task', '2024-12-11 03:52:31,242 - tno.mpc.communication.httphandlers - INFO - Server 192.168.1.1:61002 shutdown', 'Total bytes received: 1930', 'Total messages received: 5', '2024-12-11 03:52:31,246 - tno.mpc.communication.httphandlers - INFO - Client 192.168.1.2:61001 shutdown', 'Total bytes sent: 766', 'Total messages sent: 5', '2024-12-11 03:52:31,246 - tno.mpc.communication.pool - INFO - Pool shutdown.', 'Total bytes sent: 766', 'Total messages sent: 5', 'Total bytes received: 1930', 'Total messages received: 5', '\tCommand being timed: "python3.9 server_energy.py 1 912"', '\tUser time (seconds): 6.58', '\tSystem time (seconds): 0.77', '\tPercent of CPU this job got: 86%', '\tElapsed (wall clock) time (h:mm:ss or m:ss): 0:08.46', '\tAverage shared text size (kbytes): 0', '\tAverage unshared data size (kbytes): 0', '\tAverage stack size (kbytes): 0', '\tAverage total size (kbytes): 0', '\tMaximum resident set size (kbytes): 81028', '\tAverage resident set size (kbytes): 0', '\tMajor (requiring I/O) page faults: 0', '\tMinor (reclaiming a frame) page faults: 16081', '\tVoluntary context switches: 70', '\tInvoluntary context switches: 125', '\tSwaps: 0', '\tFile system inputs: 0', '\tFile system outputs: 0', '\tSocket messages sent: 0', '\tSocket messages received: 0', '\tSignals delivered: 0', '\tPage size (bytes): 4096', '\tExit status: 0', 'PowerTOP outputing using base filename log/powertop_10mbits_2024-12-11_03-53-48.csv']
2024-12-11 03:54:21,402 - DEBUG - server all values : ['modprobe cpufreq_stats failedLoaded 0 prior measurements', 'RAPL device for cpu 0', 'RAPL device for cpu 0', 'Devfreq not enabled', 'glob returned GLOB_ABORTED', 'Preparing to take measurements', 'Taking 1 measurement(s) for a duration of 11 second(s) each.', '2024-12-11 03:53:20,019 - tno.mpc.communication.httphandlers - INFO - Serving on 192.168.1.2:61001', '2024-12-11 03:53:20,726 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-11 03:53:20,834 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-11 03:53:20,861 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-11 03:53:20,906 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', 'TP is  0.001', '2024-12-11 03:53:20,946 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-11 03:53:20,951 - tno.mpc.communication.httphandlers - INFO - HTTPServer: Shutting down server task', '2024-12-11 03:53:20,953 - tno.mpc.communication.httphandlers - INFO - Server 192.168.1.2:61001 shutdown', 'Total bytes received: 766', 'Total messages received: 5', '2024-12-11 03:53:20,956 - tno.mpc.communication.httphandlers - INFO - Client 192.168.1.1:61002 shutdown', 'Total bytes sent: 1930', 'Total messages sent: 5', '2024-12-11 03:53:20,957 - tno.mpc.communication.pool - INFO - Pool shutdown.', 'Total bytes sent: 1930', 'Total messages sent: 5', 'Total bytes received: 766', 'Total messages received: 5', '\tCommand being timed: "python3.9 server_energy.py 0 912"', '\tUser time (seconds): 6.74', '\tSystem time (seconds): 0.75', '\tPercent of CPU this job got: 90%', '\tElapsed (wall clock) time (h:mm:ss or m:ss): 0:08.27', '\tAverage shared text size (kbytes): 0', '\tAverage unshared data size (kbytes): 0', '\tAverage stack size (kbytes): 0', '\tAverage total size (kbytes): 0', '\tMaximum resident set size (kbytes): 80544', '\tAverage resident set size (kbytes): 0', '\tMajor (requiring I/O) page faults: 0', '\tMinor (reclaiming a frame) page faults: 15956', '\tVoluntary context switches: 50', '\tInvoluntary context switches: 92', '\tSwaps: 0', '\tFile system inputs: 0', '\tFile system outputs: 0', '\tSocket messages sent: 0', '\tSocket messages received: 0', '\tSignals delivered: 0', '\tPage size (bytes): 4096', '\tExit status: 0', 'PowerTOP outputing using base filename log/powertop_10mbits_2024-12-11_03-53-48.csv']
2024-12-11 03:54:22,866 - DEBUG - All values from powertop for client: 1     44.9 mW  
Name: 7, dtype: object
2024-12-11 03:54:22,866 - DEBUG - Energy client: 0.333158
2024-12-11 03:54:24,192 - DEBUG - All values from powertop for server: 1       541 mW  
25     1.30 mW  
Name: 7, dtype: object
2024-12-11 03:54:24,192 - DEBUG - Energy server: 4.007597
